File: app/src/main/java/com/prox/voicechanger/ui/dialog/LoadingDialog.java
package com.prox.voicechanger.ui.dialog;

import static com.prox.voicechanger.VoiceChangerApp.TAG;

import android.content.Context;
import android.util.Log;

import androidx.annotation.NonNull;

import com.prox.voicechanger.databinding.DialogLoadingBinding;

public class LoadingDialog extends CustomDialog{

    public LoadingDialog(@NonNull Context context,DialogLoadingBinding binding) {
        super(context, binding.getRoot());
        Log.d(TAG, "LoadingDialog: create");
        setCancelable(false);
    }
}

File: app/src/main/java/com/prox/voicechanger/ui/dialog/LoadingDialog.kt
package com.prox.voicechanger.ui.dialog

import android.content.Context
import com.prox.voicechanger.VoiceChangerApp
import android.util.Log
import com.prox.voicechanger.databinding.DialogLoading2Binding

class LoadingDialog(context: Context, binding: DialogLoading2Binding) :
    CustomDialog(context, binding.root) {
    init {
        Log.d(VoiceChangerApp.TAG, "LoadingDialog: create")
        setCancelable(false)
    }
}